projects/angular/src/wizard/API.md

```diff
@@ -0,0 +1,36 @@
+# Wizard-clarity - Style API
+
+| CSS Variable Name          | Description                                                        |
+| -------------------------- | ------------------------------------------------------------------ |
+| --clr-wizard-main-bgcolor | Changes the background color in wizard element.
+| --clr-wizard-stepnav-bgcolor | Changes the background color of the side navigation bar for all wizard elements..
```

Comment:
..  -> .

Everywhere in the reviewed files, please

Reply:
Removed, and checked for other as well.